core/src/Cabana_Parallel.hpp

```diff
@@ -292,6 +297,11 @@ inline void neighbor_parallel_for(
 
     using memory_space = typename neighbor_list_traits::memory_space;
 
+    auto begin = exec_policy.begin();
+    auto end = exec_policy.end();
+    using linear_policy_type = Kokkos::RangePolicy<execution_space, void, void>;
```

Comment:
Sorry I know it is a bit late but what are these two trailing `void` template parameter there for?  Does the compile with Kokkos 3.4?

Reply:
Yeah those are unnecessary. It does pass with Kokkos 3.2, master, and develop though